modules/indexing/src/test/java/org/apache/ignite/internal/processors/query/h2/H2JavaObjectSerializerTest.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.query.h2;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+import org.apache.ignite.IgniteLogger;
+import org.apache.ignite.internal.util.typedef.internal.U;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.*;
+
+public class H2JavaObjectSerializerTest {
```

Comment:
it might pay to extend from GridCommonAbstractTest and use the log and other facilities from there. Similar to other tests in this package.

Reply:
I've updated the test to extend GridCommonAbstractTest, keeping its consistency with other tests and utilizing common test utilities.